src/dotnet/project.json

```diff
@@ -26,6 +26,8 @@
     }
   },
   "dependencies": {
+    "NuGet.CommandLine.XPlat": "3.6.0-beta.1.msbuild.1",
```

Comment:
The non-msbuild based restore command still exists to allow restoring xproj, this reference is needed for that.